### PR TITLE
Update flakiness script

### PR DESCRIPTION
**Pull request by outofambit**
- removed checking for empty responses
- use a more markdown centric output format
- group output by test ids
- use a most common set of responses as the comparison set, instead of just using the first set of responses

### CODE REVIEW

**Comment by outofambit** [comment]
@jugglinmike if you would take a look at this, that would be great! @gnarf already took a look and added a lot of great improvements via #34.

**Review by jugglinmike — CHANGES_REQUESTED** [review]
Between the need to silence npm's output and the parameterized source code, I think this script now warrants a CLI. Could you make a follow-up task for that?

File: stressor/stress-test.mts [hunk 1]
```diff
@@ -271,86 +391,187 @@ console.log(
...
+    const testCombosForTestPlan = testCombinations.filter(
+      (testCombo) => testCombo.workflowTestPlan === testPlan
     );
-  });
-  console.log(`==============================`);
+    // For each test plan, run each test combo in parallel
+    return testCombosForTestPlan.map(spawnAndCollectWorkflows);
+  }));
+
+  clearInterval(logStatusInterval);
```
[the remark this answers is not in the document]
Comment (gnarf) [reply stressor/stress-test.mts:439]: sounds good

File: stressor/stress-test.mts [hunk 2]
```diff
@@ -176,92 +239,149 @@ async function dispatchWorkflowForTestCombo(
...
+    const { testCsvRow, screenreaderResponses: baselineResponses } = compTest;
+    const differences: Array<ComparisonTestRunDifference> = [];
+    runs.forEach((run, i) => {
+      totalRows++;
+      const resultResponses =
+        run.results.findLast((l) => l.testCsvRow === compTest.testCsvRow)
+          ?.screenreaderResponses ?? [];
+      const isRowEqual =
+        JSON.stringify(resultResponses) ===
+        JSON.stringify(baselineResponses);
```
[the remark this answers is not in the document]
Comment (gnarf) [reply stressor/stress-test.mts:328]: I think it's just an Array<string>, JSON should be good for that, but I think that a better compare function is better on "memory" anyway - [`_.isEqual`](https://lodash.com/docs#isEqual) would probably nicely do the job, are you against importing that dependency, or should I embed a custom rewritten version of it?

**Comment by gnarf** [comment]
> Between the need to silence npm's output and the parameterized source code, I think this script now warrants a CLI. Could you make a follow-up task for that?

I agree - It would be nice to be able to pass the parameters / specify filenames and could use a CLI front-end script.  @outofambit and I discussed how we were getting closer and closer to needing it. cc @ccanash "Add CLI front-end to stress test script" for a future task

File: stressor/stress-test.mts (hunk 1 above)
[the remark this answers is not in the document]
Comment (gnarf) [reply stressor/stress-test.mts]: f980db3

File: stressor/stress-test.mts (hunk 2 above)
[the remark this answers is not in the document]
Comment (gnarf) [reply stressor/stress-test.mts]: f980db3 with isEqual - can always write an explicit loop `isStringArrayEqual` if needed

File: stressor/stress-test.mts (hunk 1 above)
[the remark this answers is not in the document]
Comment (gnarf) [reply stressor/stress-test.mts]: 60e6181

**Review by jugglinmike — APPROVED** [review]
> I think it's just an Array, JSON should be good for that, but I think that a better compare function is better on "memory" anyway - [`_.isEqual`](https://lodash.com/docs#isEqual) would probably nicely do the job, are you against importing that dependency, or should I embed a custom rewritten version of it?

Thanks for calling that out. I agree that JSON serialization is a safe identity for arrays of strings; I must have misinterpreted the types. I'm still glad you followed through, anyway :)

**Pull request merged**